src/app/docs/sectioned-form/demo/demo-address-form.component.html

```diff
@@ -1,28 +1,2 @@
-<div
-  class="sky-emphasized"
-  style="margin-bottom: 5px;"
->
-  <label>
-    Home
-  </label>
-</div>
-<div>
-  410 17th Street
-</div>
-<div>
-  Denver, CO 80202-4402
-</div>
-<div
-  class="sky-emphasized"
-  style="margin: 5px 0 5px 0;"
->
-  <label>
-    Vacation
-  </label>
-</div>
-<div>
-  3345 Franciscan Way
-</div>
-<div>
-  Tampa, FL 10255-0098
+<div>Address form
```

Comment:
I know these are just for visual testing and dev use - but is there a reason we are gutting these?

Reply:
Actually - these first two are the actual demo that shows up for consumers on the docs - it doesn't seem like we would want to gut those.

Reply:
@Blackbaud-TrevorBurch Gutting them b/c the content isn't representative of what it should be (i.e., a form) and weren't formatted using SKY UX components (i.e., not how one should display that kind of content). This updates moves them more to the approach for docs used in Tabs and Vertical tabs - a label just stating what would display here instead of fake content.